### PR TITLE
Docs: Refactored alert groups and silences sections

### PR DESCRIPTION
**Pull request by achatterjee-grafana**
Broke down alert groups and silences, adjusted weights of the topics and fixed broken relrefs.

### CODE REVIEW

**Review by gillesdemey — APPROVED** [review]

File: docs/sources/alerting/silences/create-silence.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
++++
+title = "Create a silence"
+description = "Add silence alert notification"
+keywords = ["grafana", "alerting", "silence", "mute"]
+weight = 450
+aliases = ["/docs/grafana/latest/alerting/unified-alerting/silences/"]
++++
+
+# Create a silence
+
```
Comment (chri2547) [inline docs/sources/alerting/silences/create-silence.md:10]: No intro or before you begin?

File: docs/sources/alerting/silences/edit-silence.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
++++
+title = "Edit silences"
+description = "Edit alerting silences"
+keywords = ["grafana", "alerting", "silence", "mute"]
+weight = 450
+aliases = ["/docs/grafana/latest/alerting/unified-alerting/silences/"]
++++
+
+# Edit silences
+
```
Comment (chri2547) [inline docs/sources/alerting/silences/edit-silence.md:10]: no intro or before you begin?

File: docs/sources/alerting/silences/remove-silence.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
++++
+title = "Remove silences"
+description = "Remove alerting silences"
+keywords = ["grafana", "alerting", "silence", "mute"]
+weight = 451
+aliases = ["/docs/grafana/latest/alerting/unified-alerting/silences/"]
++++
+
+# Remove silences
+
```
Comment (chri2547) [inline docs/sources/alerting/silences/remove-silence.md:10]: no intro or before you begin?

**Review by chri2547 — COMMENTED** [review]
See comments

File: docs/sources/alerting/silences/create-silence.md (hunk 1 above)
Comment (achatterjee-grafana) [reply docs/sources/alerting/silences/create-silence.md:10]: Not really - No useful information to add, that I can think of. Suggestions?

File: docs/sources/alerting/silences/edit-silence.md (hunk 2 above)
Comment (achatterjee-grafana) [reply docs/sources/alerting/silences/edit-silence.md:10]: I am not sure what information will be useful here. Suggestions?

File: docs/sources/alerting/silences/remove-silence.md (hunk 3 above)
Comment (achatterjee-grafana) [reply docs/sources/alerting/silences/remove-silence.md:10]: I am not sure what information will be useful here. Suggestions?

**Pull request merged**